lib/shopify_app/carrier_services_manager_job.rb

```diff
@@ -0,0 +1,10 @@
+module ShopifyApp
+  class CarrierServicesManagerJob < ActiveJob::Base
+    def perform(shop_domain:, shop_token:)
+      ShopifyAPI::Session.temp(shop_domain, shop_token) do
+        manager = CarrierServicesManager.new
+        manager.create_carrier_services
```

Comment:
I think this should be `recreate_carrier_services!` so that on each auth we are making sure the shop has what we need. That being said destroying and creating sucks. We should look into making recreate smarter so that it checks to make sure what we want is there